site-metadata.js

```diff
@@ -115,4 +115,14 @@ siteMetadata.urls.navLinks = [
   ...siteMetadata.urls.navCommunityLinks,
 ];
 
+siteMetadata.urls.addonsLearnLinks = [
+  { title: 'How to install addons', icon: 'book', href: '' },
+  {
+    title: 'Create an addon',
+    icon: 'edit',
+    href: 'https://www.chromatic.com/blog/how-to-build-a-storybook-addon/',
+  },
+  { title: 'Add to the catalog', icon: 'add', href: '' },
```

Comment:
No href?

Reply:
Yea, these are placeholders for now. Just building out the UI first while we wait for the back-end and the docs to be updated.